src/lwe/iwlib.h

```diff
@@ -27,10 +27,31 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
-#include <netdb.h>		/* gethostbyname, getnetbyname */
+#include <netdb.h>		    /* gethostbyname, getnetbyname */
 #include <net/ethernet.h>	/* struct ether_addr */
 #include <sys/time.h>		/* struct timeval */
 #include <unistd.h>
+#include <stdint.h>
+
+/*
+ * This are types for compatibily
+ */
+#ifdef __linux__
+  #include "linux/types.h"
+  #include <net/if_arp.h>		 /* For ARPHRD_ETHER */
+#else
+
+typedef unsigned char  u_char;
```

Comment:
this stuff shouldnt be necessary. it's defined in sys/types.h when _GNU_SOURCE is defined.
see https://github.com/t6x/reaver-wps-fork-t6x/pull/89/commits/bd357b580de8cbaf1f3cf008fbd64de9964ffd63